upgrade/UPGRADE-v11.0.0-dev.md

```diff
@@ -830,6 +830,19 @@ There you can find links to upgrade notes for other versions too.
     - see #project-base-diff to update your project
     - replace parameter `kernel.root_dir` with `%kernel.project_dir%/src` in your codebase
     - parameter `shopsys.framework.javascript_sources_dir` was removed because it's not used anywhere
+- debug data collectors are now feature compatible with Symfony 5 ([#2484](https://github.com/shopsys/shopsys/pull/2484))
```

Comment:
should it be really "feature" or "future"/"forward"? Maybe it is a phrase, I am just not familiar with it :slightly_smiling_face:

Reply:
you're absolutely right... it's future compatibility 😆  it was probably a long day. Fixed everywhere